yellowbrick/utils.py

```diff
@@ -155,6 +157,45 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+#From here: http://stackoverflow.com/questions/26248654/numpy-return-0-with-divide-by-zero
+def numpy_div0( a, b ):
+    """
+    Ufunc-extension that returns 0 instead of nan when dividing numpy arrays
+    example: >> div0( [-1, 0, 1], 0 ) -> [0, 0, 0]
+    """
+    with np.errstate(divide='ignore', invalid='ignore'):
+        c = np.true_divide( a, b )
+        c[ ~ np.isfinite( c )] = 0  # -inf inf NaN
+    return c
+
+
+def find_text_color(base_color, text_colors=("black","white")):
```

Comment:
Good point on breaking out the tuple. It was because I was at first imagining that the function might be designed to take more than 2 text colors, but it seemed like a 'middle' color would probably never really be used so didn't implement it. 

I'd vote for keeping the `_color` part of the variable definition to help keep it clear what type of variable type is expected. `dark` and `light` can be any valid matplotlib color definition (so `"black"` works but so does a hex code or rgb).